api/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# FROM golang:latest AS builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest AS builder
```

Comment:
Need to use the same UBI image as other files 8.X (IIRC 8.10 refer: https://github.com/project-kessel/relations-api/blob/main/Dockerfile#L1)